internal/cmd/cmd.go

```diff
@@ -198,48 +212,68 @@ func (c *Cmd) Start() error {
 	stdin, stdout, stderr := p.Stdio()
 	if c.Stdin != nil {
 		// Do not make stdin part of the error group because there is no way for
-		// us or the caller to reliably unblock the c.Stdin read when the
-		// process exits.
+		// us to reliably unblock the c.Stdin read when the process exits.
+		// Even if `stdin` is closed, the runtime can block indefinitely on reading
+		// c.Stdin, so the only reliable way to unblock this is with:
+		// c.Stdin.CloseWrite() (if it implements it) or c.Stdin.Close().
+		// However, we are only passed on the Reader end of Stdin, and closing the
+		// upstream c.Stdin breaks with the functionality that os.exec.Cmd implements.
```

Comment:
What is this comment trying to justify?

Reply:
It expands on the original comment with more info, and gives context as to what is happening.
I didnt change anything here, so no justification.